client/dashboard/sites/settings/index.tsx

```diff
@@ -26,10 +26,6 @@ export default function SiteSettings( { siteSlug }: { siteSlug: string } ) {
 	const { data: site } = useSuspenseQuery( siteBySlugQuery( siteSlug ) );
 	const { data: settings } = useQuery( siteSettingsQuery( site.ID ) );
 
-	if ( ! settings ) {
```

Comment:
Nice. We should remove this line then (I think):

https://github.com/Automattic/wp-calypso/blob/45f077cf6e11c1c1a6c47d92eb504efa3f93c2c9/client/sites/settings/v2/router.tsx#L60

Reply:
We can continue to preload the data, but ensure it doesn’t block rendering.

Reply:
Ohh you're right!